05a-run_ica.py

```diff
@@ -14,51 +14,55 @@
 
 import config
 
-# Here we always process with the 1 Hz highpass data (instead of using
-# l_freq) because ICA needs a highpass.
 
-
-grads_rejection_limit = 4000e-13
-mags_rejection_limit = 4e-12
 decim = 11  # do not touch this value unless you know what you are doing
 
-
 def run_ica(subject, tsss=config.mf_st_duration):
-    print("Processing subject: %s%s"
-          % (subject, (' (tSSS=%d)' % tsss) if tsss else ''))
+    print("Processing subject: %s" % subject)
+
     meg_subject_dir = op.join(config.meg_dir, subject)
-    raw_fnames = op.join(meg_subject_dir, '%s_audvis_filt_sss_raw.fif' % subject)
+
+    epochs_fname = op.join(meg_subject_dir,
+                           config.base_epochs_fname.format(**locals()))
+    
+    epochs_for_ica_fname = op.splitext(epochs_fname)[0] + '_for_ica.fif'
+
+    epochs_for_ica = mne.read_epochs(epochs_for_ica_fname, preload=True)
```

Comment:
I would recompute the epochs here and not rely on epochs that have been saved elsewhere.
It's fast to recompute and avoids storing big files just to get a tiny -ica.fif file.

Reply:
Ok, we can do that. I was thinking it is good to do it in one script to keep the epochs for ICA as similar to the 'real' ones as possible. But since all specifications come from config, this should still work.